presentation-compiler/test/dotty/tools/pc/tests/hover/HoverTypeSuite.scala

```diff
@@ -134,15 +134,15 @@ class HoverTypeSuite extends BaseHoverSuite:
          |class C
          |object Foo:
          |    extension [T](using A)(s: T)(using B)
-         |        def double[G](using C)(times: G) = (s.toString + s.toString) * times
+         |        def double[G <: Int](using C)(times: G) = (s.toString + s.toString) * times
```

Comment:
Actually, why the change to the test here? Could you possibly ping me or anyone from the Metals team next time something within the presentation compiler needs to change? That would be greatly appreciated :pray:

Reply:
Sure.  It's because otherwise `double` doesn't typecheck, because `String * Any` doesn't typecheck.

Reply:
Ach, right! Makes sense!